src/store/actions.js

```diff
@@ -149,29 +148,33 @@ export default {
     commit('deleteItem', item.id);
     dispatch('_setItemForEditting', null);
   },
-  _setCurrentItems({ commit }, items) {
-    commit('setCurrentItems', items);
-  },
   _setItemForEditting({ commit }, item) {
     commit('setItemForEditting', item);
   },
 
-  // changing statuses
+  // visualization
 
-  _changeChangingListStatus({ commit }, status) {
-    commit('changeChangingListStatus', status);
+  _shuffleFilteredList({ commit }) {
+    commit('shuffleFilteredList');
   },
+  _setSorting({ commit }, sorting) {
+    commit('setSorting', sorting);
 
-  // visualization modes
-
-  _switchInvertMode({ commit }) {
-    commit('switchInvertMode');
+    if (sorting === 'default') {
+      commit('setMode', 'list');
+    }
   },
-  _switchCloudMode({ commit }) {
-    commit('switchCloudMode');
+  _setMode({ commit }, mode) {
+    commit('setMode', mode);
+
+    if (mode === 'cloud' || mode === 'stars') {
+      commit('setSorting', 'shuffle');
+    } else {
+      commit('setSorting', 'default');
```

Comment:
and it also can be ternary in passing second argument

Reply:
No problem, but it's much less obvious